debian/changelog

```diff
@@ -1,6 +1,10 @@
 matrix-synapse-py3 (1.47.0+nmu1) UNRELEASED; urgency=medium
 
   * Update scripts to pass Shellcheck lints.
+  * Remove unused Vagrant scripts from debian/ directory.
+  * Change package Architecture to any.
+  * Preinstall the "wheel" package when building virtualenvs.
+  * Do not error if /etc/default/matrix-synapse is missing.
 
  -- root <root@cae79a6e79d7>  Fri, 22 Oct 2021 22:20:31 +0000
```

Comment:
I'm not sure how this line  got through review in the first place, but please could you give it a proper name? it does end up in the final changelog.

Reply:
Probably arose from quickly hacking things together in an ephemeral Debian container; will fix in a followup PR in just a sec.